titus-client/src/main/java/com/netflix/titus/runtime/endpoint/v3/grpc/GrpcJobManagementModelConverters.java

```diff
@@ -388,20 +389,27 @@ private static BasicContainer toCoreBasicContainer(com.netflix.titus.grpc.protog
     }
 
     private static Volume toCoreVolume(com.netflix.titus.grpc.protogen.Volume grpcVolume) {
-        return Volume.newBuilder()
-                .withName(grpcVolume.getName())
-                .withSharedContainerVolumeSource(toCoreSharedVolumeSource(grpcVolume.getSharedContainerVolumeSource()))
-                .build();
+        if (grpcVolume.getSharedContainerVolumeSource() != null) {
```

Comment:
I think this should be oneof in protobuf model. I assume that each volume will be of exactly one type?

Reply:
Added https://github.com/Netflix/titus-api-definitions/pull/150